nntrainer/optimizers/lr_scheduler_exponential.h

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file   lr_scheduler_exponential.h
+ * @date   09 December 2021
+ * @brief  This is Exponential Learning Rate Scheduler class
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Parichay Kapoor <pk.kapoor@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+
+#ifndef __LEARNING_RATE_SCHEDULER_EXPONENTIAL__
+#define __LEARNING_RATE_SCHEDULER_EXPONENTIAL__
+#ifdef __cplusplus
+
+#include <string>
+
+#include <lr_scheduler_constant.h>
+
+namespace nntrainer {
+
+/**
+ * @class   Constant Learning Rate Scheduler class
```

Comment:
Constant => exponential

Reply:
Added in later PR, so I will skip here for faster merge.